Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/TelescopeDetectorOptions.hpp

```diff
@@ -0,0 +1,20 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Utilities/Options.hpp"
```

Comment:
This include is not needed anymore.

Reply:
It's still necessary because we have the `range_range`.

Reply:
But it only needs to be included in the `.cpp` file.

Reply:
Absolutely. Fixed.